src/fittable_models.jl

```diff
@@ -0,0 +1,59 @@
+# Convenience function for wavelength conversion
+aa_to_invum(wave::Real) = 10000 / wave
+aa_to_invum(wave::Quantity) = aa_to_invum(ustrip(u"angstrom", wave))
+
+"""
+    FM90(C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)(λ::Real)
+    FM90(C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)(λ::Quantity)
+
+### Parameters
+* `C1` - y-intercept of linear term
+* `C2` - slope of liner term
+* `C3` - amplitude of 2175 Å bump
+* `C4` - amplitude of FUV rise
+* `xo` - centroid of 2175 Å bump
+*  `γ` - width of 2175 Å bump
+
+Fitzpatrick & Massa (1990) 6 parameter ultraviolet shape model, this model is only applicable at UV wavelengths.
+
+If `λ` is a `Unitful.Quantity` it will be automatically converted to Å and the
+returned value will be `UnitfulAstro.mag`.
+"""
+@with_kw struct FM90{T <: Number}
+    C1::T = 0.10
+    C2::T = 0.70
+    C3::T = 3.23
+    C4::T = 0.41
+    xo::T = 4.60
+    γ::T = 0.99
+
+    function FM90{T}(C1::T, C2::T, C3::T, C4::T, xo::T, γ::T) where T <: Number
+        xo < 0 && error("Invalid axis xo=$xo. xo must be greater than or equal to 0")
+        γ < 0 && error("Invalid axis γ=$γ. γ must be greater than or equal to 0")
+        new{T}(C1, C2, C3, C4, xo, γ)
+    end
+end
```

Comment:
this can be tidied up (from [Parameters.jl manual](https://mauro3.github.io/Parameters.jl/stable/manual/))

I've also found the derivation in the original paper. The parameters they use are pretty simple- there are four terms, each with a coefficient. there is a constant term (c1), a linear term (c1 * x), a Drude profile (c3 * D(x; gamma, x0)) and a "curvature" term (c4 * F). Given this, as well as the consideration that we don't want to force our users into using special unicode characters (like \gamma as a keyword arg), I think we should parametrize the struct with
`c1,c2,c3,c4,x0,gamma`.


```julia
@with_kw struct FM90{T<:Number} @deftype T
    c1 = 0.10
    c2 = 0.70
    c3 = 3.23
    c4 = 0.41
    x0 = 4.60
    gamma = 0.99
    @assert x0 \geq 0 "`x0` must be \geq 0, got $x0"
    @assert gamma \geq 0 "`gamma` must be \geq 0, got $gamma"
end
```